css/styles.css

```diff
@@ -50,15 +52,82 @@ li {
   background-color: #e5e5e5;
 }
 
+.close-menu {
```

Comment:
- [ ] The mobile menu needs to follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, margins), you can start by adding some blur to the background